test/domain/reaction/remove.spec.ts

```diff
@@ -0,0 +1,52 @@
+
+import { describe, expect, it } from 'vitest';
+import { COMIC_REACTION_ID, COMIC_RECORD_TYPE, COMMENT_REACTION_ID, COMMENT_RECORD_TYPE, DELETED_REACTION_ID, EXISTING_REQUESTER, REACTION_COMIC_ID, REACTION_COMMENT_ID, REACTION_RECORD_TYPE, ReactionNotFound, SOME_OTHER_REQUESTER, createDatabase, remove } from './_fixtures/reaction.fixture';
+
+describe('domain/reaction/remove', () =>
+{
+    it('should soft delete a reaction with a comment', async () =>
+    {
+        const database = await createDatabase();
+
+        await remove(EXISTING_REQUESTER, COMMENT_REACTION_ID);
+
+        const reaction = await database.readRecord(REACTION_RECORD_TYPE, COMMENT_REACTION_ID);
+        expect(reaction.deleted).toBe(true);
+
+        // only soft delete the reaction, not the comment
+        const comment = await database.readRecord(COMMENT_RECORD_TYPE, REACTION_COMMENT_ID);
+        expect(comment.deleted).toBeUndefined();
```

Comment:
I don't think we should test the things we haven't implemented. Maybe we can place a comment in the delete function that we've chosen to only soft-delete the reaction, and not its children.